databases/week2/Part4.sql

```diff
@@ -0,0 +1,27 @@
+USE dc;
```

Comment:
I know you created however you have to mention the created database statement before USE it, exp:
CREATE DATABASE dc;